taprpc/mintrpc/mint.proto

```diff
@@ -187,6 +187,14 @@ message MintAsset {
     a BIP-86 key will be derived from the underlying wallet.
     */
     taprpc.ScriptKey script_key = 12;
+
+    /*
+    Precision is a single integer value that represents how fine grained an
+    asset unit is. The value of this field is the number of decimal places that
+    this asset can be divided into. For example, if the precision is 2, a unit
```

Comment:
Thought we were going with an approach where the supply amount stays, and this is mainly for UI purposes? So a re-write here should be: 
> If the precision is 2, an a user has 1000 units of an asset, then this would be displayed as 10.00

Reply:
How would we ever deal with sub-units then? it would always be fixed to `.00`

Reply:
There are no sub units. It's always a whole unit. Integer arhietmetic everywhere. A whole unit may be part of a denomination however.

Reply:
It also wouldn't be fixed to `.00`. If I have 999 unit, then that's `9.99`.